draft-jholland-quic-multicast.md

```diff
@@ -878,6 +886,8 @@ Further, the use of multicast channels likely requires increased coordination be
 
 For large deployments, server implementations will often need to operate on separate devices from the ones generating the multicast channel packets, and will need to be designed accordingly.
 
+As several MC_ACKs can be bundled for efficiency purposes, servers SHOULD make sure that packets are stored and able to be retransmitted for a reasonable time. This SHOULD be at least the max_ack_delay of a channel plus half the RTT between client and server.
```

Comment:
IIRC packets are not retransmitted, though the information in frames can be, if I read 13.3 of 9000 right. Should we copy text from the ACK handling there, and maybe also refer to 13.2 of 9000 for how it's handled if the ack frame buffer gets too large?

Reply:
Agreed, I am now wondering if we should just point to 13.2 in general and not elaborate here at all since the behavior we want seems to be pretty much in line with what's written there as far as I can tell.